cloudbridge/cloud/providers/gce/resources.py

```diff
@@ -1670,7 +1670,7 @@ def description(self, value):
 
     @property
     def size(self):
-        return self._volume.get('sizeGb')
+        return int(self._volume.get('sizeGb'))
```

Comment:
@chiniforooshan Is this guaranteed to be non-null? We had a few problems with OpenStack or AWS where some of these properties return null values and that results in an exception...

Reply:
I've not seen a case where this is null. If it's OK with you, I prefer it breaks if it's null so that we see it, try to understand when it happens, and fix it properly.